scripts/check_test_list.py

```
@@ -523,23 +529,63 @@ def validate_test_lists(test_lists_dir: str, test_base_dir: str):
+        __file__).resolve().parent.parent / "jenkins" / "ci_versions.properties"
+    with open(props_file) as f:
+        for line in f:
+            line = line.strip()
+            if line.startswith("TRT_TEST_DB_VERSION="):
+                return line.split("=", 1)[1]
+    raise RuntimeError(f"TRT_TEST_DB_VERSION not found in {props_file}")
+
+
+def install_python_dependencies(llm_src: str) -> None:
+    """Install Python deps for collection — no TRT-LLM wheel or compile."""
     subprocess.run(f"cd {llm_src} && pip3 install -r requirements-dev.txt",
```

Ruff (0.16.1)

[error] `subprocess` call with `shell=True` identified, security issue

(S602)